src/Makefile.am

```diff
@@ -33,6 +33,8 @@ endif
 if ENABLE_NEURON
   sources += nccl_ofi_interface_neuron.c
 else
+  AM_CPPFLAGS += $(CUDA_CPPFLAGS)
```

Comment:
but you added CUDA_CPPFLAGS and CUDA_LDFLAGS into CPPFLAGS and LDFLAGS?

Reply:
Ignore this; i missed that you updated CPPFLAGS but the code previously removed it later.